vm/src/builtins/set.rs

```diff
@@ -637,13 +679,18 @@ impl PySet {
 
     #[pymethod(magic)]
     fn ior(zelf: PyRef<Self>, set: AnySet, vm: &VirtualMachine) -> PyResult<PyRef<Self>> {
+        // fn ior(zelf: PyRef<Self>, set: PyObjectRef, vm: &VirtualMachine) -> PyResult<PyRef<Self>> {
         zelf.inner.update(set.into_iterable_iter(vm)?, vm)?;
+        // zelf.inner.update_internal(set.into_object(), vm)?;
+        // zelf.inner.update_internal(set, vm)?;
```

Comment:
If you have a mind to fix `ior`, if `PyObjectRef` cannot be converted into `AnySet` we should return `PyArithmeticValue::NotImplemented`